### PR TITLE
Prettier Import Sort Plugin

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

We currently use `import/order` to sort imports. I hate this for a couple reasons:

- This feels like formatting, which is what prettier is for. ESLint is for catching functional or code issues, not formatting.
- I don't run ESLint auto-fixing automatically, and IMO folks generally shouldn't need to because it really shouldn't be catching much that's super auto-fixable, because only formatting is easily auto-fixable without supervision.
- I want to move to a world where we don't have anything yell at you locally for prettier because ideally you just set that up and don't think about it, and it can fail in CI, but ESLint / TS will yell at you in your editor, keeping your signal to noise ratio high, which is good.
- I just personally hate import sorting and this will at least make me think about it less.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by hayes-mysten — APPROVED** [review]

**Review by williamrobertson13 — APPROVED** [review]
stamping for democracy 🎏

File: prettier.config.js [hunk 1]
```diff
@@ -8,6 +8,16 @@ module.exports = {
 	tabWidth: 2,
 	trailingComma: 'all',
 	useTabs: true,
+	plugins: ['@ianvs/prettier-plugin-sort-imports'],
+	importOrder: [
+		'<BUILT_IN_MODULES>',
+		'<THIRD_PARTY_MODULES>',
+		'',
```
Comment (hayes-mysten) [inline prettier.config.js:15]: what do the blank groups do?

Comment (hayes-mysten) [reply prettier.config.js:15]: looks like it creates an empty line between groups

Comment (Jordan-Mysten) [reply prettier.config.js:15]: Yeah it creates a newline between groups. Kind of weird, but 🤷

**Pull request merged**